airflow-core/src/airflow/api_fastapi/core_api/routes/public/hitl.py

```diff
@@ -190,10 +203,37 @@ def get_hitl_detail(
         task_id=task_id,
         session=session,
         map_index=map_index,
+        try_number=None,
     )
     return task_instance.hitl_detail
 
 
+@task_instances_hitl_router.get(
+    task_instance_hitl_path + "/tries/{try_number}",
+    status_code=status.HTTP_200_OK,
+    responses=create_openapi_http_exception_doc([status.HTTP_404_NOT_FOUND]),
+    dependencies=[Depends(requires_access_dag(method="GET", access_entity=DagAccessEntity.HITL_DETAIL))],
+)
+def get_hitl_detail_try(
```

Comment:
Nit:
```suggestion
def get_hitl_try_details(
```

To be consistant with `get_task_instance_try_details`

Reply:
then it probably should be `get_hitl_detail_try_details` instead

Reply:
Double `detail` in the name is confusing. I get why it's there though. 

Also I think in the current impl, you missed the trailing `s` `get_hitl_detail_try_detail`